cards/src/test/kotlin/com/decathlon/vitamin/compose/cards/utils/VitaminCardsPrimaryTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.decathlon.vitamin.compose.cards.utils
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import app.cash.paparazzi.Paparazzi
+import com.decathlon.vitamin.compose.cards.VitaminCards
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(TestParameterInjector::class)
+class VitaminCardsPrimaryTest {
+    @get:Rule
+    val paparazzi = Paparazzi()
+
+    @Suppress("LongMethod")
+    @Test
+    fun medium(@TestParameter theme: Theme) {
```

Comment:
because we have only 1 size, you could make 1 test by Figma variant, like you did in the Sample, to make design reviews easier
-topImage
-fullImageSolid
-fullImage
-sideImage
-withoutImage

Reply:
Are we sure about that ? Because this would mean testing code that is not managed by the component itself. This also implies to add picture un the component only for test purposes.

Reply:
We can let this for now. You're right about testing only what we provide